### PR TITLE
Admin view summary bill

### PR DESCRIPTION
**Pull request by phucynwa**

### CODE REVIEW

**Comment by phucynwa** [comment]
ready

**Comment by phucynwa** [comment]
https://dev.framgia.com/issues/184391

File: app/views/admin/bills/index.html.erb [hunk 1]
```diff
@@ -19,4 +19,32 @@
...
+            <option value="4">4</option>
+            <option value="5">5</option>
+            <option value="6">6</option>
+            <option value="7">7</option>
+            <option value="8">8</option>
+            <option value="9">9</option>
+            <option value="10">10</option>
+            <option value="11">11</option>
+            <option value="12">12</option>
+          </select>
```
Comment (mdmahmudurrahman) [inline app/views/admin/bills/index.html.erb]: You can use bootstrap date/month picker

**Comment by mdmahmudurrahman** [comment]
commented

**Comment by phucynwa** [comment]
ready

File: app/assets/javascripts/custom.js [hunk 2]
```diff
@@ -18,4 +18,9 @@ $(document).on('turbolinks:load', function () {
     window.print();
     $('#print-bill').siblings().show();
   });
+  $(".datepicker").datepicker({
+    format: "mm-yyyy",
+    viewMode: "months",
+    minViewMode: "months"
```
Comment (mdmahmudurrahman) [inline app/assets/javascripts/custom.js]: Em, single quote in JS

**Comment by mdmahmudurrahman** [comment]
commented

**Comment by phucynwa** [comment]
ready

**Pull request merged**